### ISSUE TITLE
jQuery 3.0.0 deferred exception

### ISSUE THREAD

**Issue by teekey99** [stevenwanderski/bxslider-4#1022]
bxSlider stucks at loading while using with jQuery 3.0.0. Console throws following exception: 

`jQuery.Deferred exception: Cannot read property 'indexOf' of undefined TypeError: Cannot read property 'indexOf' of undefined.` 

Console shows `undefined` value at jQuery's init function. Stack trace:

```
jQuery.Deferred exception: Cannot read property 'indexOf' of undefined TypeError: Cannot read property 'indexOf' of undefined
    at jQuery.fn.load (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:9612:12)
    at HTMLImageElement.<anonymous> (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:10331:40)
    at Function.each (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:359:19)
    at jQuery.each (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:152:17)
    at HTMLImageElement.<anonymous> (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:10330:12)
    at Function.each (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:359:19)
    at jQuery.each (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:152:17)
    at loadElements (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:10327:50)
    at setup (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:10317:7)
    at init (http://localhost/ixprint-presentation/www/js/ixprint-presentation.js?v=1466078325:10224:7) undefined
```

All scripts are bundled using gulp v3.9.1 with gulp-concat v2.6.0. If I use previous stable version of jQuery (2.2.4) bxSlider works without any issues. I've used bxSlider on many web presentations and this is the very first occurence of such issue in my case. Issue tested in latest versions of Chrome and Firefox on Ubuntu 16.04.

**Comment by pradeekyahvi** [comment]
yes i got the same issue, 

`jquery-3.0.0.min.js:2 jQuery.Deferred exception: Cannot read property 'createDocumentFragment' of undefined TypeError: Cannot read property 'createDocumentFragment' of undefined
    at oa (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery-3.0.0.min.js:3:5915)
    at Ha (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery-3.0.0.min.js:3:15067)
    at r.append (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery-3.0.0.min.js:3:16723)
    at r.fn.(anonymous function) [as appendTo] (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery-3.0.0.min.js:3:18269)
    at Questionare.init (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery.nupoll.js:25:32)
    at new Questionare (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery.nupoll.js:16:14)
    at r.$.fn.questionare (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery.nupoll.js:47:9)
    at HTMLDocument.<anonymous> (http://localhost:63342/tuts_jquery_plugin_devops/index.html?:10:22)
    at j (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery-3.0.0.min.js:2:29588)
    at k (http://localhost:63342/tuts_jquery_plugin_devops/js/jquery-3.0.0.min.js:2:29902) undefined`

**Comment by atomantic** [comment]
a little more callstack info:

**notifyjs**
```javascript
selector.find('img:not([src=""]), iframe').each(function() {
        $(this).one('load error', function() {
          if (++count === total) { callback(); }
        }).each(function() {
          if (this.complete) { $(this).load(); }
        });
      });
```
`$(this).load();` triggers
**jquery 3**
```javascript
/**
 * Load a url into a page
 */
jQuery.fn.load = function( url, params, callback ) {
	var selector, type, response,
		self = this,
		off = url.indexOf( " " );
```
where `url`, `params` and `callback` are all undefined.

**Comment by atomantic** [comment]
FYI, patch is here: https://github.com/stevenwanderski/bxslider-4/pull/1024

**Comment by teekey99** [comment]
Thanks for the heads-up! I'll close this issue once the pull request is approved and merged.
